packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -882,85 +882,108 @@ async function handler(req: NextApiRequest & { userId?: number | undefined }) {
     triggerEvent: WebhookTriggerEvents.MEETING_ENDED,
   };
 
-  const subscribersMeetingEnded = await getWebhooks(subscriberOptionsMeetingEnded);
-
-  subscribersMeetingEnded.forEach((subscriber) => {
-    if (rescheduleUid && originalRescheduledBooking) {
-      cancelScheduledJobs(originalRescheduledBooking);
-    }
-    if (booking && booking.status === BookingStatus.ACCEPTED) {
-      scheduleTrigger(booking, subscriber.subscriberUrl, subscriber);
-    }
-  });
+  try {
+    const subscribersMeetingEnded = await getWebhooks(subscriberOptionsMeetingEnded);
 
-  // Send Webhook call if hooked to BOOKING_CREATED & BOOKING_RESCHEDULED
-  const subscribers = await getWebhooks(subscriberOptions);
-  console.log("evt:", {
-    ...evt,
-    metadata: reqBody.metadata,
-  });
-  const bookingId = booking?.id;
-
-  const eventTypeInfo: EventTypeInfo = {
-    eventTitle: eventType.title,
-    eventDescription: eventType.description,
-    requiresConfirmation: eventType.requiresConfirmation || null,
-    price: stripeAppData.price,
-    currency: eventType.currency,
-    length: eventType.length,
-  };
+    subscribersMeetingEnded.forEach((subscriber) => {
+      if (rescheduleUid && originalRescheduledBooking) {
+        cancelScheduledJobs(originalRescheduledBooking);
+      }
+      if (booking && booking.status === BookingStatus.ACCEPTED) {
+        scheduleTrigger(booking, subscriber.subscriberUrl, subscriber);
+      }
+    });
+  } catch (error) {
+    log.error("Error while running scheduledJobs for booking", error);
```

Comment:
Just a doubt, shouldn't this be console.error("...")

Reply:
It's an internal logger we use sometimes.
`const log = logger.getChildLogger({ prefix: ["[api] book:user"] });`